src/accumulator.jl

```diff
@@ -7,9 +7,10 @@ end
 ## constructors
 
 Accumulator{T, V}() where {T,V<:Number} = Accumulator{T,V}(Dict{T,V}())
+Accumulator(map::AbstractDict) = Accumulator(Dict(map))
 
 counter(T::Type) = Accumulator{T,Int}()
-counter(dct::Dict{T,V}) where {T,V<:Integer} = Accumulator{T,V}(copy(dct))
+counter(dct::AbstractDict{T,V}) where {T,V<:Integer} = Accumulator{T,V}(Dict(dct))
```

Comment:
```suggestion
counter(dct::AbstractDict{T, V}) where {T, V<:Integer} = Accumulator{T, V}(Dict(dct))
```

Reply:
See also #539

Reply:
i think no space is the dominant style in the file here (except that one case above on L9).

Reply:
Lets make the change it gets close to the goal of #539

Reply:
let's leave style changes to a different PR (happy to see an issue for it!)